lib/main_app.rb

```diff
@@ -9,19 +9,23 @@ def run
     @game = GameLogic.new
     @console_output = ConsoleOutput.new
 
-    until @game.is_there_winner? do
+    until @game.board_full? || @game.is_there_winner? do
```

Comment:
Can we change this to ?
```suggestion
    until @game.is_over? do
```
and then add that method to GameLogic

Reply:
👍🏽